test-app/build-tools/jsparser/tests/jasmine.json

```diff
@@ -1,7 +1,7 @@
 {
-    "spec_dir": "tests/specs",
+    "spec_dir": "specs",
```

Comment:
nitpick: I don't think we were using the `spec_dir` ever since we rewrote the javascript tests, I think it's safe to just drop this one.

Reply:
I haven't gotten around to enabling the tests, I was thinking on fixing these test for a later PR. For now I just moved them to the folder of jsparser so we don't forget them